src/components/ContextBar.js

```diff
@@ -133,7 +134,8 @@ export const ContextBar = ({
           tooltipSide: 'left',
           disabled: !isTerminalEnabled,
           href: terminalLaunchLink,
-          onClick: window.location.hash === terminalLaunchLink && currentRuntime?.status === 'Stopped' ? () => startCurrentRuntime() : undefined,
+          onClick: window.location.hash === terminalLaunchLink && currentRuntime?.status === 'Stopped' ? () => startCurrentRuntime() : Ajax().Metrics.captureEvent(Events.analysisLaunch,
+            { origin: 'contextBar', source: 'terminal', application: 'terminal', workspaceName, namespace }),
```

Comment:
Could you run the formatter? It should reformat the two lines above as below:

```suggestion
          onClick: window.location.hash === terminalLaunchLink && currentRuntime?.status === 'Stopped' ?
            () => startCurrentRuntime() :
            Ajax().Metrics.captureEvent(Events.analysisLaunch,
              { origin: 'contextBar', source: 'terminal', application: 'terminal', workspaceName, namespace }),
```

Reply:
When I clicked on the terminal icon on the context bar while having a stopped runtime, I noticed that the new tab that opens up has the legacy layout (currently in `prod`):

![image](https://user-images.githubusercontent.com/5438223/155623811-0800a647-a0a4-4d9d-b3f7-02c94104c47d.png)

and if the user uses the RuntimeManager on that tab to resume their runtime, we would end up not collecting the metric even though the user had attempted to launch terminal on the new tab. Am I missing something?

Reply:
hmm... I just refactored this whole block so its more clear. I feel like we have a tendency to over-use ternary statements and avoid the dreaded if, but oftentimes if makes it a lot clearer 😂

Reply:
Sure, when `if` is more readable and gets the job done, no need to be dogmatic about it.